metricbeat/module/system/raid/raid.go

```diff
@@ -67,33 +69,67 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 		return nil, err
 	}
 
+	sysMountPoint := filepath.Join(config.MountPoint, "/sys/block")
+
 	return &MetricSet{
 		BaseMetricSet: base,
 		fs:            fs,
+		sysfs:         sysMountPoint,
```

Comment:
It can be a bit error-prone to call this sysfs when it contains the path to `/sys/block` and not to `/sys`.